deployments/tls-app.yaml

```diff
@@ -27,11 +11,10 @@ spec:
       labels:
         app: tls-app
     spec:
-      serviceAccountName: certificate-init
       initContainers:
         - name: certificate-init-container
-          image: gcr.io/hightowerlabs/certificate-init-container:0.0.1
-          imagePullPolicy: Always
+          image: repo.lab.ppops.net/dev-docker-local/certificate-init-container:latest
```

Comment:
I guess it is OK for now, But we should switch to start using actual version numbers instead of latest once we have everything working end to end

Reply:
We need to get builds to Docker Hub working first.